src/screens/Roles/Roles.tsx

```diff
@@ -28,6 +29,20 @@ const Roles = () => {
     setComponentLoader(false);
   }, []);
 
+  useEffect(() => {
+    if (searchByName !== '') {
+      refetch({
+        filter: searchByName,
+      });
+    } else {
+      if (count !== 0) {
```

Comment:
You need to write tests for this

Reply:
That code was added to avoid infinite rendering when the component is first loaded ( since I was using useEffect with searchByName input.

How do I write tests for that? Can I please get some ideas?

Reply:
Ask the talawa-admin channel on slack